finetuning/generalists/training/light_microscopy/obtain_lm_datasets.py

```diff
@@ -0,0 +1,116 @@
+import os
```

Comment:
I am confused why these files are part of this PR, they should have been merged via #228 already!

Reply:
Ah I might know the reason. Probably because I created a branch from `lm-generalist` branch, hence it also took those new files into consideration (which at that moment weren't a part of master)